projects/plutus/config/afe_cfg.h

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#define PLUTUS_CFG_AFE_SPI_PORT SPI_PORT_1
+#define PLUTUS_CFG_AFE_SPI_BAUDRATE 750000
+#define PLUTUS_CFG_AFE_SPI_MOSI \
+  { .port = GPIO_PORT_A, .pin = 7 }
+#define PLUTUS_CFG_AFE_SPI_MISO \
+  { .port = GPIO_PORT_A, .pin = 6 }
+#define PLUTUS_CFG_AFE_SPI_SCLK \
+  { .port = GPIO_PORT_A, .pin = 5 }
+#define PLUTUS_CFG_AFE_SPI_CS \
+  { .port = GPIO_PORT_A, .pin = 4 }
+
+// Use normal mode
+#define PLUTUS_CFG_AFE_MODE LTC_AFE_ADC_MODE_7KHZ
+
+// Using all 12 cell inputs
+#define PLUTUS_CFG_INPUT_BITSET_FULL 0xFFF
+// Using 6 cell inputs split across the 2 muxes
+#define PLUTUS_CFG_INPUT_BITSET_SPLIT (0x7 << 6 | 0x7)
+// Using first 6 cell inputs
+#define PLUTUS_CFG_INPUT_BITSET_FIRST 0x3F
+
+#ifdef PLUTUS_CFG_DEBUG
```

Comment:
Good idea on debug macros. Can we just have a global `DEBUG=1` environment variable/flag in the Makefile for it though? A lot of boards could use this. Also a warning when flashing that a debug build is being flashed would be good. That way we can catch if someone didn't rebuild from scratch if they built some stuff with `DEBUG=1` and others with `DEBUG=0`

Reply:
Yeah, global flags can be wonky with rebuilds. The only reason I really do this is because I don't want to deal with allocating arrays. Otherwise, I'd just use an enum or something.

Supporting a global debug flag would be pretty easy. Warning would be difficult because someone could've built a file with `DEBUG=1` and then flashed with `DEBUG=0`.